src/osd/PeeringState.h

```diff
@@ -265,8 +265,13 @@ class PeeringState : public MissingLoc::MappingInfo {
     virtual uint64_t get_snap_trimq_size() const = 0;
 
     /// Send cluster message to osd
+    #if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
```

Comment:
Perhaps `&& !defined(WITH_ALIEN)` is unnecessary. IIRC it was a solution to build some parts to of the common static library to satisfy dependencies of `BlueStore` hosted in the alien world.